docs/UserGuide.md

```diff
@@ -380,7 +388,7 @@ Edit the information of an existing elderly or volunteer in FriendlyLink, based
 
 Edits an existing elderly based on their index in the elderly list.
 
-Format: `edit_elderly INDEX [n/NAME] [ic/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_DATE] [re/REGION] [r/RISK_LEVEL] [t/TAG]… [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…`
+Format: `edit_elderly <INDEX> [n/NAME] [ic/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_DATE] [re/REGION] [r/RISK_LEVEL] [t/TAG]… [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…`
```

Comment:
Perhaps you might want to add a section in terminology to mention that preamble is enclosed in angle brackets